static/js/vue-cdr-access/src/components/full_record/adminUnit.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <div class="browse-header">
+        <div class="columns">
+            <div class="column">
+                <bread-crumbs :ignore-search-state="false"
+                              :object-path="recordData.briefObject.objectPath">
+                </bread-crumbs>
+            </div>
+        </div>
+        <div class="columns">
+            <div class="column">
+                <h2 :class="isDeleted">
+                    <thumbnail :thumbnail-data="recordData"
+                               :allows-full-access="allowsFullAuthenticatedAccess"></thumbnail>
+                    {{ recordData.briefObject.title }}
+                </h2>
+                <p>
+                    <strong>{{ $t('full_record.subjects') }}:</strong>
+                    <template v-if="hasSubjects">
+                        {{ recordData.briefObject.subject.join(', ')}}
+                    </template>
+                    <template v-else>
+                        {{ $t('full_record.no_subjects') }}
+                    </template>
+                </p>
+                <template v-if="recordData.briefObject.abstractText">
```

Comment:
It might make sense to break the display of abstracts into its own component since it's going to be the same on all 4 pages and has some very specific formatting/functionality. Should make it easier to test too, rather than needing to test it for each object type

Reply:
Maybe we could make this another ticket?

Reply:
https://jira.lib.unc.edu/browse/BXC-4025